### PR TITLE
:memo: updated docs

### PR DESCRIPTION
**Pull request by lstuma**

### CODE REVIEW

**Comment by lstuma** [comment]
all tests passed and @raspitim is asleep, bypassing branch protections. **review tomorrow please** if possible

File: docs/pages/tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+    @app.view('/api/test', 'application/json')
+    def api_test(request):
+        return {'this data is':'autoconverted to an HTTPResponse object'}
+
+If we want to process data provided in the request, we can use the `get`, `data`, and `files` functions::
+
+    @app.view('/api/test', 'text/html')
+    def api_test(request):
+        if request.method == 'GET':
```
Comment (raspitim) [inline docs/pages/tutorial.rst:46]: replace if statements with match cases

File: docs/pages/tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    BASE_DIR = pathlib.Path(__file__).parent.resolve()
+
+    app = Pigeon()
+
+    # override settings from imported module
+    import mysettings
+    app.settings.override(mysettings)
+
+    # override settigns using dictionary
+    mysettings = {
```
Comment (raspitim) [inline docs/pages/tutorial.rst:111]: missing: how to pass settings dict to Pigeon object

Comment (raspitim) [reply docs/pages/tutorial.rst:111]: (override)

**Comment by raspitim** [comment]
> all tests passed and @raspitim is asleep, bypassing branch protections. **review tomorrow please** if possible

done.

File: docs/pages/tutorial.rst (hunk 1 above)
Comment (lstuma) [reply docs/pages/tutorial.rst:46]: right - we should do that!

**Comment by lstuma** [comment]
fixed in [39378f6](https://github.com/TeamSmil3y/PigeonPost/commit/39378f68df37edb578dc18bcfac65235763188c6)

**Pull request merged**